WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/FeeOrDiscountLineDetailsViewModel.swift

```diff
@@ -140,22 +142,21 @@ final class FeeOrDiscountLineDetailsViewModel: ObservableObject {
 
     /// Returns the formatted string value of a price, substracting the current stored discount entered by the merchant
     ///
-    func calculatePriceAfterDiscount(_ price: String) -> String {
-        guard let price = currencyFormatter.convertToDecimal(price) else {
-            return ""
-        }
-        let discount = NSDecimalNumber(decimal: finalAmountDecimal)
-        let priceAfterDiscount = price.subtracting(discount)
+    var recalculateFormattedPriceAfterDiscount: String {
```

Comment:
```suggestion
    var formattedPriceAfterDiscount: String {
```

Strange to have a verb in a var

Reply:
You're right, updated: https://github.com/woocommerce/woocommerce-ios/pull/14083/commits/795fcff991af23c210d64975dfcfbcb2cd1b04d7